backend/src/aggregator.rs

```diff
@@ -48,8 +51,8 @@ const THIRD_PARTY_NETWORKS_MAX_NODES: usize = 500;
 impl Aggregator {
     pub fn new(denylist: HashSet<String>) -> Self {
         Aggregator {
+            hashes: HashMap::new(),
```

Comment:
```suggestion
            genesis_hashes: HashMap::new(),
```
…or go full semantic and call this "networks"?

Reply:
I like `genesis_hashes`, I reckon it has lower gravity.